tests/dummy/config/optional-features.json

```diff
@@ -1,3 +1,6 @@
 {
-  "jquery-integration": false
+  "application-template-wrapper": false,
+  "default-async-observers": true,
+  "jquery-integration": true,
```

Comment:
i believe we want `jquery-integration` set to `false`, and also remove the jquery dep from package.json/yarn.lock. (and then make sure the dummy app/addon still runs ok.